edsdme/blocks/announcements-preview/announcements-preview.js

```diff
@@ -0,0 +1,147 @@
+import { getCaasUrl } from '../../scripts/utils.js';
+import { getConfig } from '../utils/utils.js';
+
+export default async function init(el) {
+  const config = getConfig();
+  const newestCards = [];
+
+  const block = {
+    el,
+    collectionTag: '"caas:adobe-partners/collections/announcements"',
+    ietf: config.locale.ietf,
+  };
+
+  const blockData = {
+    caasUrl: getCaasUrl(block),
+    ietf: config.locale.ietf,
+    title: '',
+    buttonText: '',
+    link: '',
+    empty: 'Currently, there are no partner announcements',
+  };
+
+  const app = document.createElement('announcements-preview');
+
+  Array.from(el.children).forEach((row) => {
```

Comment:
I don't know if we need this to be so flexible, authoring of the component is simple so we can just demand that first row of the table be a `title` etc. wdyt ?

Reply:
i disscused this with Ben and we agreed that if something is missing from table eg title or button name or link we exclude it from component, if you see any reason why we should not i can change it